topics/icrc_34_delegation.md

```diff
@@ -1,50 +1,60 @@
-# ICRC-34: Get Global Delegation
+# ICRC-34: Delegation
 
 [![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31)
 [![Extension Badge](https://img.shields.io/badge/EXTENDS-ICRC--25-ffcc222.svg)](./icrc_25_signer_interaction_standard.md)
 [![Standard Issue](https://img.shields.io/badge/ISSUE-ICRC--34-blue?logo=github)](https://github.com/dfinity/wg-identity-authentication/issues/115)
 
 <!-- TOC -->
-* [ICRC-34: Get Global Delegation](#icrc-34-get-global-delegation)
+* [ICRC-34: Delegation](#icrc-34-delegation)
   * [Summary](#summary)
   * [Method](#method)
   * [Scope (according to the ICRC-25 standard)](#scope-according-to-the-icrc-25-standard)
-    * [Example RPC Request Permission](#example-rpc-request-permission)
-  * [Request](#request)
-    * [Example RPC Request](#example-rpc-request)
-    * [Response](#response)
-    * [Example RPC Response](#example-rpc-response)
+    * [Example RPC Permission Request](#example-rpc-permission-request)
+  * [icrc-25-supported-standards](#icrc-25-supported-standards)
+  * [Request params](#request-params)
+    * [Example RPC Delegation Request](#example-rpc-delegation-request)
+  * [Result](#result)
+    * [Example RPC Delegation Response](#example-rpc-delegation-response)
   * [Message Processing](#message-processing)
   * [Errors](#errors)
 <!-- TOC -->
 
 ## Summary
 
-When a relying party wants to authenticate as a user, it uses
-a [session key](https://internetcomputer.org/docs/current/references/ic-interface-spec/#signatures), and
-below `icrc34_get_global_delegation` method to obtain a delegation chain that allows the session key to sign for the
-user's global identity. The obtained delegation chain **MUST** be restricted by the signer to canister targets that
-trust the relying party following the [ICRC-28](./icrc_28_trusted_origins.md) standard.
+When a relying party wants to avoid going through the signer and user approval for each individual call, it can use the icrc34_delegation method described below. 
 
-- Due to the canister target restriction, only the canisters that the relying party owns can be called on behalf of the user.
-- A new delegation chain must be obtained for every new canister created by the relying party.
-- The identity is the same across all relying parties, the user is not anonymous between different relying parties.
+This method generates a delegation chain, an object with a stable identifier, allowing the relying party to sign transactions on behalf of the user without having to go through the signer and user approval. This also means the signer does not need to stay connected to the signer to make calls on behalf of the user.
 
-## Method
+The delegation chain's stable identifier can be either: 
+1. **Account Delegation**: an identity that may hold tokens and other assets.
```

Comment:
That property is not exclusive to account delegations, see the current ecosystem of in-dapp wallets.

I think the crucial point is that it is stable across many relying parties (but restricted by `targets`).

And further, if it does hold assets, the delegation _must not_ give the relying party access to them.

Maybe: "Restricted access to the signers identity, such that it is stable across many relying parties but cannot be used to operate on tradable assets and shared infrastructure"